console/console-init/ui/src/AppLayout.tsx

```diff
@@ -19,8 +19,8 @@ import {
 } from "components";
 import { AppRoutes } from "Routes";
 import { AppNavHeader } from "./AppNavHeader";
-import brandImg from "./assets/images/logo.svg";
-import avatarImg from "./img_avatar.svg";
+import brandImg from "logo.svg";
```

Comment:
Can you please verify if this change is needed, for me, the console doesn't break because of assets.